### DISCUSSION TITLE
Running ani-cli from remote server

### DISCUSSION THREAD

**Discussion by EltonChing** [pystardust/ani-cli#1305] [General]
Hi,
I'm a long time user of ani-cli and I was wondering if anyone had an idea as to how one would stream ani-cli from a remote server to your current machine's media player. I installed it on my home server to try and do so. 
I'm running Debian 12.

**Comment by justchokingaround** [comment]
Need more info, does the script runs the instance of media Player on server? If yes use -> ffmpeg 

If only link fetching is needed and run the media player locally, then use -> netcat as a micro API

**Reply by EltonChing** [reply]
Let's say I had a local home server with ani-cli installed on it or any kind video stream application, how would I, after logging via ssh to said server on my windows machine, pipe that stream and output the media to VLC (for example).
